source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -280,7 +280,14 @@ Node Parameters and Usage
 
       asr_phrases
 
-    - Forwards the call to a selected phone number after optionally reading a prompt.
+      connect_timeout
+
+    - Forwards the call to a selected phone number after optionally reading a prompt. If ringing_rollover is enabled for the network, a connect_timeout value can be configured along with a "Failover" child node.
+
+  * - Failover
+    - \*destination_phone_number
+
+    - Forwards the call to a fallback destination number if the original destination configured on the parent "Connect" node returns a busy signal OR if the configured connect_timeout value on the parent "Connect" node is exceeded. This node type can only be configured as a child of "Connect" nodes.
```

Comment:
> This node type can only be configured as a child of "Connect" nodes.

This isn't necessarily accurate given that `ivr_ringing_rollover_extensions` has been made, right?

Reply:
Valid point, I just thought we were waiting to document `ivr_ringing_rollover_extensions`. I can remove the `"Connect"` reference and just have it say "parent node". I can also modify the last statment to say "This node type can only be configured as a child, and not as a standalone node." Thoughts?

Reply:
Sounds good 👍